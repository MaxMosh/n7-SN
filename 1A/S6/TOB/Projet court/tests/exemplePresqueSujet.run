java allumettes.Jouer Xavier@humain Ordinateur@rapide <<EOF
2
X
5
3
1
EOF

