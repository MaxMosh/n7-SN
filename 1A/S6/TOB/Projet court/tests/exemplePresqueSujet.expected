Allumettes restantes : 13
Xavier, combien d'allumettes ? Xavier prend 2 allumettes.

Allumettes restantes : 11
Ordinateur prend 3 allumettes.

Allumettes restantes : 8
Xavier, combien d'allumettes ? Vous devez donner un entier.
Xavier, combien d'allumettes ? Xavier prend 5 allumettes.
Impossible ! Nombre invalide : 5 (> 3)

Allumettes restantes : 8
Xavier, combien d'allumettes ? Xavier prend 3 allumettes.

Allumettes restantes : 5
Ordinateur prend 3 allumettes.

Allumettes restantes : 2
Xavier, combien d'allumettes ? Xavier prend 1 allumette.

Allumettes restantes : 1
Ordinateur prend 1 allumette.

Ordinateur perd !
Xavier gagne !
